js/w3c/headers.js

```diff
@@ -155,6 +156,18 @@ define(
                     ret += ", <span class='ed_mailto'><a class='u-email email' " + rm + " href='mailto:" + p.mailto + "'>" + p.mailto + "</a></span>";
                 }
                 if (p.note) ret += " (" + p.note + ")";
+                if (p.extras && p.extras.length !== 0) {
+                    ret += ", ";
+                    for (var k = 0; k < p.extras.length; k++) {
+                        var extra = p.extras[k];
+                        if (extra.name) {       
+                            ret += extra.label ? "<span class='" + extra.label + "'>" : "<span>";
+                            ret += extra.href ? "<a" + rpe + " href='" + extra.href + "'>" + extra.name + "</a>" : extra.name;
```

Comment:
> On 30 Sep 2015, at 01:09 , Marcos Caceres <notifications@github.com> wrote:
> 
> In js/w3c/headers.js <https://github.com/w3c/respec/pull/508#discussion_r40742437>:
> 
> > @@ -155,6 +156,18 @@ define(
> >                      ret += ", <span class='ed_mailto'><a class='u-email email' " + rm + " href='mailto:" + p.mailto + "'>" + p.mailto + "</a></span>";
> >                  }
> >                  if (p.note) ret += " (" + p.note + ")";
> > +                if (p.extras && p.extras.length !== 0) {
> > +                    ret += ", ";
> > +                    for (var k = 0; k < p.extras.length; k++) {
> > +                        var extra = p.extras[k];
> > +                        if (extra.name) {
> > +                            ret += extra.label ? "<span class='" + extra.label + "'>" : "<span>";
> > +                            ret += extra.href ? "<a" + rpe + " href='" + extra.href + "'>" + extra.name + "</a>" : extra.name;
> I'm a bit worried about extra.name as a XSS opportunity... need to check if we have a sanitize function.
> 

Changed it. I just followed the pattern:-)
> 


----
Ivan Herman, W3C
Digital Publishing Lead
Home: http://www.w3.org/People/Ivan/
mobile: +31-641044153
ORCID ID: http://orcid.org/0000-0003-0782-2704